### PR TITLE
Update contribution steps and info on frame conversions

### PR DESCRIPTION
**Pull request by TSC21**

### CODE REVIEW

File: mavros/README.md [hunk 1]
```diff
@@ -176,13 +185,19 @@ With that tool you may place mavlink package in your mavros workspace.
...
+2. Clone the repo (`git clone https://github.com/mavlink/mavros.git`);
+3. Create a remote connection to your repo (`git remote add <remote_repo> git@github.com:<YourGitUser>/mavros.git`);
+4. Create a feature/dev branch (`git checkout -b <feature_branch>`);
+5. Add the changes;
+6. Apply the changes by commiting (`git commit -m "<message>"` or `git commit -a` and then write message; if adding new files: `git add <path/to/file.ext>`);
+7. Check code style `uncrustify -c ${ROS_WORKSPACE}/mavros/mavros/tools/uncrustify-cpp.cfg --replace --no-backup <path/to/file.ext>`;
+8. Fix small code style errors and typos;
+9. Run tests:
+ - with `catkin_make`, issue `catkin_make tests` and then `catkin_make run_tests`;
+ - with `catkin tools`, issue `catkin build --catkin-make-args run_tests`;
```
Comment (vooon) [inline mavros/README.md]: Nope, `catkin run_tests`

File: mavros/README.md [hunk 2]
```diff
@@ -50,9 +50,18 @@ Note: ids from URL overrides ids given by system\_id & component\_id parameters.
...
+MAVROS does translate Aerospace NED frames, used in FCUs to ROS ENU frames and vice-versa.
+Rules are descrided as follows:
 
+| Transform:             |                      |
+|:----------------------:|:--------------------:|
+| Position (local frame) | `(x,y,z)`→`(y,x,-z)` |
+
+| Transform:                     | Euler angles:         | Quaternions:              |
+|:------------------------------:|:---------------------:|:-------------------------:|
+| Orientation (body-fixed frame) | `(r,p,y)`→`(r,-p,-y)` | `(w,x,y,z)`→`(x,-y,-z,w)` |
```
Comment (vooon) [inline mavros/README.md]: That not true anymore. We do PI rotation around X (ROLL) axis, not swap & change signs.
Also there no difference in transform for angles or position. Earlier position transformation is equal to PI ROLL & PI/2 YAW.

Comment (TSC21) [inline mavros/README.md]: A rotation around Roll, applied to a rotation matrix, means that r,p,y -> r,-p-y. So this is true.

Comment (vooon) [inline mavros/README.md]: Look at position table too. That note for whole diff section.
And that minuses not descriptive, better to say what we do rotation around roll.

**Pull request merged**